rosbag2_compression/package.xml

```diff
@@ -12,10 +12,13 @@
   <depend>rcpputils</depend>
   <depend>rcutils</depend>
   <depend>rosbag2_storage</depend>
+  <depend>zstd_vendor</depend>
+  <depend>zstd</depend>
```

Comment:
according to [this](https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml#L3915-L3922) I think `zstd` is the wrong rosdep key.